### DISCUSSION TITLE
Mouse input lag.

### DISCUSSION THREAD

**Discussion by lemmih** [slint-ui/slint#583] [Q&A]
Is the input lag inherent to the opengl backend or can it be improved?

Here's an example (running at 1/3 speed) where the circle position is set to the mouse position:

https://user-images.githubusercontent.com/1443107/137472343-5f47d799-55da-4ff2-83c2-048713d080f5.mov

**Comment by ogoffart** [comment]
With the Qt backend this cannot be reproduced, but I can reproduce it with the GL backend.

So it seems the problem is somewhere in the GL stack. Maybe @tronical have a better idea on where the problem lies and if that's fixable.
Could be anywhere in `winit`, `glutin` or `femtovg` or in our backend.


[Reproducer](https://sixtyfps.io/releases/0.1.3/editor/?snippet=%0Aexport+Demo+%3A%3D+Window+%7B%0A++++ta+%3A%3D+TouchArea+%7B++%7D%0A++++Rectangle+%7B%0A++++++++background%3A+red%3B%0A++++++++width%3A+20px%3B%0A++++++++height%3A+20px%3B%0A++++++++border-radius%3A+width+%2F+2%3B%0A++++++++x%3A+ta.mouse-x+-+width+%2F+2%3B%0A++++++++y%3A+ta.mouse-y+-+height+%2F+2%3B%0A++++%7D%0A%7D%0A)

**Reply by tronical** [reply]
I've created #596 - this seems like a bug to me.
